docs: Search Metadata Batch 05 (#14621)
hashicorp/consul: website/content/docs/k8s/operations/tls-on-existing-cluster.mdx

---
layout: docs
page_title: Rolling Updates to TLS for Existing Clusters on Kubernetes
description: >-

> Consul's Helm chart began supporting TLS communication within clusters in version 0.16.0. Follow the instructions to trigger rolling updates for consul-k8s without causing downtime.

Reviewer: ```suggestion
Consul Helm chart 0.16.0 and later supports TLS communication within clusters. Follow the instructions to trigger rolling updates for consul-k8s without causing downtime.
```
